frontends/ol-components/src/components/Card/ListCard.tsx

```diff
@@ -0,0 +1,200 @@
+import React, {
+  FC,
+  ReactNode,
+  Children,
+  ImgHTMLAttributes,
+  isValidElement,
+} from "react"
+import styled from "@emotion/styled"
+import { theme } from "../ThemeProvider/ThemeProvider"
+import { Link } from "react-router-dom"
+import { Wrapper, containerStyles } from "./Card"
+
+const LinkContainer = styled(Link)`
+  ${containerStyles}
+  display: flex;
+
+  :hover {
+    text-decoration: none;
+    border-color: ${theme.custom.colors.silverGrayLight};
+    box-shadow:
+      0 2px 4px 0 rgb(37 38 43 / 10%),
+      0 2px 4px 0 rgb(37 38 43 / 10%);
+    cursor: pointer;
+  }
+`
+
+const Container = styled.div`
+  ${containerStyles}
+`
+
+const Content = () => <></>
+
+const Body = styled.div`
+  flex-grow: 1;
+  overflow: hidden;
+  margin: 24px;
+  ${theme.breakpoints.down("md")} {
+    margin: 12px;
+  }
+
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+`
+
+const Image = styled.img`
+  display: block;
+  width: 236px;
+  height: 122px;
+  margin: 24px 24px 24px 0;
+  border-radius: 4px;
+  object-fit: cover;
+  ${theme.breakpoints.down("md")} {
+    width: 111px;
+    height: 104px;
+    margin: 0;
+    border-radius: 0;
+  }
+
+  background-color: ${theme.custom.colors.lightGray1};
+  flex-shrink: 0;
+`
+
+const Info = styled.div`
+  ${{ ...theme.typography.subtitle3 }}
+  margin-bottom: 16px;
+  ${theme.breakpoints.down("md")} {
+    ${{ ...theme.typography.subtitle4 }}
+    margin-bottom: 8px;
+  }
+
+  color: ${theme.custom.colors.silverGrayDark};
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+`
+
+const Title = styled.h3`
+  flex-grow: 1;
+  text-overflow: ellipsis;
+  ${{ ...theme.typography.subtitle1 }}
+  height: ${theme.typography.pxToRem(40)};
+  ${theme.breakpoints.down("md")} {
+    ${{ ...theme.typography.body3 }}
+    height: ${theme.typography.pxToRem(32)};
+  }
+
+  overflow: hidden;
+  margin: 0;
+
+  @supports (-webkit-line-clamp: 2) {
```

Comment:
I saw you used `TruncateText` elsewhere. Should we use that here, possibly adding the `@supports` to it?

Reply:
Yes!.. pushed